src/peft/tuners/poly/router.py

```diff
@@ -0,0 +1,65 @@
+from abc import ABC, abstractmethod
+
+import torch
+from torch import nn
+from torch.distributions.relaxed_bernoulli import RelaxedBernoulli
+
+from .config import PolyConfig
+
+
+EPS = 1e-12
+
+
+def get_router(poly_config: PolyConfig) -> nn.Module:
+    if poly_config.poly_type == "poly":
+        return PolyRouter(poly_config)
+    else:
+        raise NotImplementedError
```

Comment:
Can we raise a `ValueError` with a helpful error message here?

Reply:
sure, that would be helpful